### PR TITLE
Created Quick Start doc

### PR DESCRIPTION
**Pull request by kevPo**
This is the first of 2 PRs to address #339. 

When/if this merges I'll create the second PR to update the main README.md like the issue requested. I didn't want to create a link to a page that doesn't exist yet. 😎

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Quick Start
+
+Yarn is required to create a basic structure. (node: ">=12", yarn: ">=1.15")
```
Comment (thedavidprice) [inline docs/quick-start.md:3]: Ok, we should elaborate on this just a bit more. Can borrow from the [Tutorial Prereq](https://redwoodjs.com/tutorial/prerequisites#nodejs-and-yarn-versions) section if needed (we'll just keep it shorter).
- both Yarn and Node.js are required at given versions or more recent
- Link to Yarn: https://classic.yarnpkg.com/en/docs/install/
- Link to Node.js: https://nodejs.org/en/

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+```
```
Comment (thedavidprice) [inline docs/quick-start.md:4]: I suggest introducing this step a bit more, e.g. Run the following command to create a new Redwood project in a "redwoodblog" folder within your current directory.

File: docs/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  yarn create redwood-app ./redwoodblog
+```
+Start the development server
```
Comment (thedavidprice) [inline docs/quick-start.md:7]:
```suggestion
Start the development server:
```

**Review by thedavidprice — CHANGES_REQUESTED** [review]
Thanks for getting this started @kevPo! The nav item and setup look great. I've made some comments inline to improve the copy and instructions a bit.

I also suggest the following:
1. Add a section "## The Redwood CLI"
    - quick copy to introduce the fact that Redwood developer experience relies heavily on the command line
    - can see all commands with `yarn rw --help` (note: rw is alias of redwood). Not sure if you should copy+paste the output of that in the doc or not. It might be helpful, but it might also be a wall of code that's one wall too many... ¯\_(ツ)_/¯
    - and can dig in further to a command for more options, e.g. `yarn rw db help`
    - link to the CLI Doc for all commands and options: https://redwoodjs.com/docs/cli-commands
2. (Maybe) Add a section "## Redwood's Architecture"
    - this is definitely a brainstorm — a lot of people don't initially understand how React, GraphQL, and Prisma connect to the Web, API, and DB aspects of a Redwood project. This forum topic, with drawing and links, can be a big "ah ha!" for people. So...
        - maybe at a minimum link to this Forum topic
        - maybe add the "Links to Docs and Tutorial" (minus Netlify info)
        - or ...? I'll defer to you
3. Add a section "## Next Steps"
    - Copy + link to the Tutorial: https://redwoodjs.com/tutorial/welcome-to-redwood
    - Invitation to join the community via:
        - Forums: https://community.redwoodjs.com
        - Chat: https://discord.gg/redwoodjs

**Comment by kevPo** [comment]
Thanks for all the information @thedavidprice! I don't claim to be so great at creating copy so I'd be more than happy to adjust anything as you wish. 

I took a shot at creating all the sections, with the exception of the architecture section. I was wondering if that might make more sense as a stand alone section in docs?

File: docs/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+What exactly just happened? Glad you asked.
+
+- `yarn rw db save` creates a snapshot of our UserExample model for our migration
+- `yarn db up` applies the migration and creates a new table in our database called `UserExample`
+- `yarn rw generate scaffold UserExample` tells Redwood to create the necessary Pages, SDL, and Services for the given Model 
+
+Just like that, we are done. No seriously. Visit http://localhost:8910/user-examples to see for yourself. 
+
+<img src="/images/emptyUserExample.png" />
```
Comment (thedavidprice) [inline docs/quick-start.md:61]:
```suggestion
<img width="463" alt="Screen Shot 2020-10-21 at 6 28 08 PM" src="https://user-images.githubusercontent.com/2951/96807389-3eede900-13cb-11eb-828a-52210cd67e3e.png">
```
add image

File: docs/quick-start.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+yarn rw db --help
+```
+
+Take a visit to the [CLI Doc](https://redwoodjs.com/docs/cli-commands.html) to see detailed information on all commands and options.
+
+## Generators
+
+Redwood generators make monotonous developer tasks a breeze. Creating all the boilerplate code required for CRUD operations on a model can be accomplished with a few commands. Three to be exact. 
+
+Every new Redwood project comes with a default Model called UserExample in `schema.prisma`.
```
Comment (thedavidprice) [inline docs/quick-start.md:36]:
```suggestion
Every new Redwood project comes with a default Model called UserExample in `api/prisma/schema.prisma`. 
```
add path to file

**Comment by thedavidprice** [comment]
@kevPo this is great! I made two small changes including adding a different image and using github as the static image hosting provider ¯\_(ツ)_/¯

Great work and thank you! 🚀

Two (potential) Next steps:
1. Update the main introduction README.md section to simply be a link to this doc in redwoodjs.com
2. If you're up for it, I’d love to see this be a link a the end of the `yarn create redwood-app ...` installation. There are two reference PRs to see how to update the code and manage colors. See https://github.com/redwoodjs/redwood/pull/1195 and then recently https://github.com/redwoodjs/redwood/pull/1373 
    - I think this could make a great link after the line "Enter the directory and run ...". So instead of (another section), maybe this is a standalone line to "get up and running fast with this Quick Start guide..." (I definitely welcome other ideas/suggestions you might have.)

Again, no pressure on these. Please do let me know, however, so I can move things forward otherwise.

**Comment by kevPo** [comment]
Thanks @thedavidprice!

As far as next steps, I've already started working on the first one, and I'd be glad to work on the second one as well.

**Comment by thedavidprice** [comment]
@kevPo nice! Go for it. And don't hesitate to open a draft PR with bare-bones code to get started. 

Thanks in advance.

**Comment by Tobbe** [comment]
I think we have some CSS leaking from the roadmap to the quick start guide (The "Generators" link styling)

image

@kevPo Do you want to have a look at that? No worries if you don't. I can create a new issue for it if you don't want to try to fix it. Or I can even just try to fix it myself. Just let me know, k? 🙂

**Comment by kevPo** [comment]
@Tobbe I'll take a look. Might be a few hours before I can get to it. Thanks for letting me know.

**Comment by kevPo** [comment]
@thedavidprice @Tobbe It's looking like changing the title of that section might be the quickest way to fix this issue. I'm thinking we could go with either `## Powerful Generators` or `## Redwood Generators`. 

I was originally thinking about using `## The Power of Generators`, but it felt weird be directly under the section `## The Redwood CLI`. A lot of "The" in there.

Do you all have a preference? Or if you'd like me to fix this is another fashion, I'd be glad to do that as well.

**Comment by Tobbe** [comment]
> Do you all have a preference? Or if you'd like me to fix this in some other fashion, I'd be glad to do that as well.

I'd like to make it so that those header styles only apply to the roadmap doc. And a way to do that would be to move that `styles` map in `docutron.js` to `roadmap.js` and pass it to `create` as part of the options object.

Please create a new PR with that fix, and tag me and @jtoar as he wrote the `roadmap.js` script and added the styles object.

**Pull request merged**